src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const partner = (person.sex === 'f')
+    ? "husband's"
+    : "wife's";
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      <p className="Person__age">
+        {person.age ? `I am ${person.age}` : null}
+      </p>
```

Comment:
do you need a `p` with a `null` value? wrap this whole element with `age &&` condition